src/commands/channels/rename.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as MixFlags from '../../utils/flags'
+import * as ChannelsAPI from '../../mix/api/channels'
+import {DomainOption} from '../../utils/validations'
+import {MixClient, MixResponse} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+import {Output} from '@oclif/parser/lib/flags'
+import {ChannelsRenameParams} from '../../mix/api/channels-types'
```

Comment:
sort so move right aftert `import * as...`

Reply:
I've organized the imports, but let's discuss a standard for how imports should be sectioned and organized.